subprojects/docs/src/docs/release/notes.md

```diff
@@ -28,6 +28,10 @@ Gradle now suppoprts version range in parent elements of POM, which was introduc
 IMPORTANT: if this is a patch release, ensure that a prominent link is included in the foreword to all releases of the same minor stream.
 Add-->
 
+### Eclipse plugin separates output folders
+
+The `eclipse` plugin now defines separate output directories for each source folders. Also, each source folder and dependency defines an additional `gradle_source_sets` classpath attribute. The attribute specifies to which sourceSet the entry belonged. Future [Buildship](http://eclipse.org/buildship) versions will use this information to separate source sets when launching Java applications within Eclipse.
```

Comment:
❌ This section is outdated. It does not reflect the actual implementation. How about not going into this detail at all and just explaining what it does? I had a draft of that on [my branch](https://github.com/gradle/gradle/tree/oehme/eclipse/scopes).

Reply:
I added the text from your branch.